ush/forecast_postdet.sh

```diff
@@ -599,14 +600,18 @@ data_out_GFS() {
           $NCP $file $memdir/RESTART/$file
         done
       fi
-    elif [ $CDUMP = "gfs" ]; then
-      $NCP $DATA/input.nml $ROTDIR/${CDUMP}.${PDY}/${cyc}/atmos/
+    elif [ $CDUMP = "gfs" ] || [ ${RUN} = "gefs" ]; then
+      $NCP $DATA/input.nml ${ROTDIR}/${RUN}.${PDY}/$cyc/${mem:-""}/atmos/
```

Comment:
In this change, I use ${RUN}.${PDY} instead of ${CDUMP}.${PDY} based on the NCO's standard. I also add the ${mem:-""} to deal with GEFS data.